runelite-client/src/main/java/net/runelite/client/plugins/timersandbuffs/TimersAndBuffsPlugin.java

```diff
@@ -1044,6 +1080,13 @@ public void onGameTick(GameTick event)
 			}
 		}
 
+		final int regionID = WorldPoint.fromLocalInstance(client, client.getLocalPlayer().getLocalLocation()).getRegionID();
```

Comment:
Are there any teleports which leave you in the same region? (does the dueling ring land you inside that region, or just outside of it?) If not, we could remove this counter on GameState -> LOADING instead of checking each tick.

Reply:
That's a good shout! I tested it and created https://github.com/runelite/runelite/pull/17727
It turns out that the westernmost tiles are outside the region, while the other ones are in the same region as the colosseum. Since the regionID for the instanced and non-instanced version of the Colosseum is the same, I added an ``isInInstancedRegion()`` check.
I assume teleporting to the same regionID but going from the instanced to the non-instanced version would not proc ``GameState.LOADING``, based on your commment? I sadly don't have the colosseum teleport unlocked and I can't think of any other similar regions with a teleport, so I can't test it out myself.